.ls-lint.yml

```diff
@@ -0,0 +1,11 @@
+---
+ls:
+  .md: screamingsnakecase
```

Comment:
Are you sure about this one, I thought none GitHub related .md files were also going to be snake_case?

Do you mean to suggest that they should all just be SCREAMING_SNAKE_CASE.md

Reply:
README.md
CHANGELOG.md
CODE_OF_CONDUCT.md
CONTRIBUTION.md

I don't think we will have much Markdown files.

I mean if we are adding something in a folder. It would be a README.md file, no?

We can use this for now, and talk about next time we will face the feedback of this linter